shared/actions/search.js

```diff
@@ -31,50 +32,47 @@ type RawResult = {
   }
 }
 
-function parseFullName (rr: RawResult): string {
-  if (rr.keybase && rr.keybase.full_name) {
-    return rr.keybase.full_name
-  } else if (rr.service && rr.service.full_name) {
-    return rr.service.full_name
-  }
-
-  return ''
-}
-
 function parseExtraInfo (platform: ?SearchPlatforms, rr: RawResult, isFollowing: (username: string) => boolean): ExtraInfo {
-  const fullName = parseFullName(rr)
-  const serviceName = rr.service && rr.service.service_name && capitalize(rr.service.service_name)
+  const serviceName = rr.service && capitalize(rr.service.service_name || '')
 
   if (platform === 'Keybase') {
-    if (serviceName) {
+    if (rr.service) {
```

Comment:
really this code cares about the structure of the incoming rawResult objet and not a piece of derived data. if we have a service sub object we want to do this flow. if capitalize() happened to have something like if (service_name === twitter) return '' then we'd do the wrong thing (for example)